azure-eventhubs-eph/src/main/java/com/microsoft/azure/eventprocessorhost/EventProcessorOptions.java

```diff
@@ -112,11 +114,21 @@ public int getPrefetchCount() {
     /***
      * Sets the prefetch count for the underlying event hub client.
      *
-     * The default is 300. This controls how many events are received in advance. 
+     * The default is 500. This controls how many events are received in advance.
```

Comment:
Default up at the top of this class should be changed to match.

Reply:
Good catch, we will actually need to keep the same default for EPH library and update the document.

Reply:
This will be addressed in a new pull request to the dev branch.